static/js/admin/page.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Admin Page.
+ */
+
+import axios from "axios";
+import _ from "lodash";
+import React, { useState } from "react";
+import { Button } from "reactstrap";
+
+export function Page(): JSX.Element {
+  const [loading, setLoading] = useState(Boolean);
```

Comment:
is this a new syntax for `useState`? Because shouldn't the state type be declared like `useState<boolean>()`

Reply:
Good catch, there were totally off!